README.md

```diff
@@ -1,2 +1,57 @@
 # notifications-email-provider-stub
-Stub calls to our email providers for use during load testing
+
+## What this does
+During load testing, instead of sending requests to AWS SES to send emails we want to stub them out so that we don't incur unnecessary costs.
+
+When the API tries to send an email, instead of using Boto and talking to AWS SES, you can use the `AwsSesStubClient` to send an HTTP POST request to this stub app to mimic the behaviour of SES. 
+
+When this stub app receives a POST request from the API to send an email, it will
+- put a task on the ses-callback queue with a fake SES callback 
+- respond with a fake `messageID` to be used as the reference from SES
+
+
+## Set up
+
+To run locally
+
+```
+pip install -r requirements.txt
+make run
+```
+Then visit http://localhost:6301/ to see it running
+
+## Deploy
+
+Should only be deployed to preview and staging environments. It will not work on a local development environment because we do not support SES callbacks on local environments (there is no Lambda function or SQS queue to process them).
```

Comment:
pretty sure if you set `QUEUE_PREFIX` to be your prefix as you defined in your environment.sh, it'd work fine.

sorry, think i misunderstood you when you asked in slack the other day. we don't support _actual_ ses callbacks because we don't have a lambda. but there is a queue, and a stub can put tasks on the queue just fine.

Reply:
Ah, yes that makes sense. Thanks